Conjugar/UIView+Accessibility.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+extension UIView {
+    func setAccessibilityLabelInSpanish(_ label: String, region: String = Current.settings.region.accent) {
+        setAccessibilityLabel(label, spokenInLanguage: "es_\(region)")
+    }
+    
+    private func setAccessibilityLabel(_ label: String, spokenInLanguage languageCode: String) {
+        let attributes: [NSAttributedString.Key : Any] = [
+            .accessibilitySpeechLanguage: languageCode,
```

Comment:
Can remove the trailing comma. Some developers prefer to include trailing commas in order to avoid future diffs, but I prefer to avoid them for æsthetic reasons.

Reply:
Done.